plancton/__init__.py

```diff
@@ -210,18 +211,19 @@ def _read_conf(self):
       self.conf["max_docks"] = 0
     if not isinstance(self.conf["docker_cmd"], list):
       self.conf["docker_cmd"] = self.conf["docker_cmd"].split(" ")
-    if self.conf["influxdb_url"] and not "#" in self.conf["influxdb_url"]:
-      # Invalid InfluxDB URL: disable monitoring
-      self.conf["influxdb_url"] = None
+    tmp_url_list = []
```

Comment:
Ok, sounds good, as long as we remember to write on config file:
`# YAML also has a set type, which looks like this:
set:
    ? url1#...
    ? url#...
    ? url#...`